src/Xamarin.Android.Cecil/Mono.Cecil.overrides

```diff
@@ -0,0 +1,8 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+	<AssemblyName>$(AssemblyName.Replace('Mono', 'Xamarin.Android'))</AssemblyName>
+  </PropertyGroup>
+	<ItemGroup>
```

Comment:
Ah, good catch. This files was moved from monodroid repo. It is good opportunity to fix the indentation, done.